ops/dataset-copy.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
+# a script to copy a dataset from one deployment to another
+
+set -eu
+
+### INPUTS ###
+DATASET_UUID_TO_COPY="00000000-0000-0000-0000-000000000000"
+AUTH_TOKEN="authorization: Bearer ya29"
+
+SRC_URL="https://jade.datarepo-dev.broadinstitute.org"
+DST_URL="https://jade-fb.datarepo-dev.broadinstitute.org/"
+
+### utility functions ###
+token_check() {
+    RESULT=$(curl -s -X GET "${SRC_URL}/api/resources/v1/profiles" -H "accept: application/json" -H "${AUTH_TOKEN}")
```

Comment:
Could pass `-w "%{http_code}" -o /dev/null` to `curl` so that it only prints the response code, meaning you wouldn't have to match on the full response contents.

Reply:
This is neat and more appropriate. I switched it.